simulation/run_simulation.py

```diff
@@ -92,7 +92,7 @@ def run_simulation(settings):
 
     # Initialize a "guess" speed array
     driving_hours = simulation_model.get_driving_time_divisions()
-    input_speed = np.array([30] * driving_hours)
+    input_speed = np.array([60] * driving_hours)
```

Comment:
Maybe we want this default "guess" for our starting speed to be a param, this depends on how impactful this is on our ability to converge.